src/hotspot/os_cpu/bsd_aarch64/vmStructs_bsd_aarch64.hpp

```diff
@@ -31,22 +31,9 @@
 // constants required by the Serviceability Agent. This file is
 // referenced by vmStructs.cpp.
 
-#define VM_STRUCTS_OS_CPU(nonstatic_field, static_field, unchecked_nonstatic_field, volatile_nonstatic_field, nonproduct_nonstatic_field, c2_nonstatic_field, unchecked_c1_static_field, unchecked_c2_static_field) \
-                                                                                                                                     \
-  /******************************/                                                                                                   \
-  /* Threads (NOTE: incomplete) */                                                                                                   \
-  /******************************/                                                                                                   \
-  nonstatic_field(OSThread,                      _thread_id,                                      OSThread::thread_id_t)             \
-  nonstatic_field(OSThread,                      _unique_thread_id,                               uint64_t)
```

Comment:
Does the SA actually use these?  I don't see any SA changes in this patch.  If it doesn't, we should remove it until there is someone that might add support for this.  We're not enhancing the SA at this point.

Reply:
I think the SA is using these. There are references to them in the various SA files.

Reply:
Ok.  I see now why you don't also have to change SA.